fury/tests/test_actors.py

```diff
@@ -1338,11 +1338,11 @@ def test_billboard_actor(interactive=False):
 def test_sdf_actor(interactive=False):
     scene = window.Scene()
     scene.background((1, 1, 1))
-    centers = np.array([[2, 0, 0], [0, 2, 0], [0, 0, 0]]) * 11
-    colors = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
-    directions = np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1]])
-    scales = [1, 2, 3]
-    primitive = ['sphere', 'ellipsoid', 'torus']
+    centers = np.array([[2, 0, 0], [0, 2, 0], [0, 0, 0], [2, 2, 0]]) * 11
+    colors = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1], [1, 1, 0]])
+    directions = np.array([[0, 1, 0], [1, 0, 0], [0, 0, 1], [1, 1, 0]])
+    scales = [1, 2, 3, 4]
```

Comment:
Not sure why, Looking into it

Reply:
Fixed.

Reply:
There are still problems when large values are used as a `scale`. In the case of the `superellipsoid`, the actor won't be rendered. And, in the case of the `capsule` primitive, this is what you'll see when you put large `scale` values.

<video controls src="https://user-images.githubusercontent.com/9929496/113746290-2735c300-96d4-11eb-9993-fda460cb624e.mp4">

Reply:
I'll look into this

Reply:
Hi @guaje, this issue also occurs with the torus and sphere actors.
I believe all shader based actors are rendered inside a cube like structure. Giving an actor a very large size may cause conflicts with this external cube.
Do you have any thoughts on this?

Reply:
Thank you for noticing this!

First of all, can you raise an issue adding videos/gifs?

Second, I'll have to test this myself, but my initial guess is that this could be due to the fact that the scaling is managed at the shader level and not at the python level as occurs with the `billboard` actor where we scale up and down the square we use as a canvas. Then we normalize the size of the canvas, so the SDF primitive takes up as much space as possible making the coding homogeneous and easy. I can point you to the parts of the code where we do this if you're interested in trying this approach.

Reply:
I'd be happy to take a look at this issue. Thank you.